.github/workflows/upload-docs.yml

```diff
@@ -0,0 +1,48 @@
+name: Upload Docs Android
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - 'test-doc-*'
+
+jobs:
+  upload-docs:
+
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+    - name: Generate docs
+      run: |
+        ./gradlew :PrebidMobile:combinedJavadoc
+        cp -r PrebidMobile/build/combinedJavadoc docs_new
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: docs
+        clean: false
+    - name: Commit documentation changes
+      run: |
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Where did this 41898282+github-actions email come from? I guess it's fine, just not familiar with it.

Reply:
It's an email for GitHub actions bot. [Ref](https://github.com/marketplace/actions/hexo-action#refer-anchor-1).